### PR TITLE
Remote Widgets synchronization

### PR DESCRIPTION
**Pull request by jesusbotella**
This PR implements remote widgets synchronization, same as #62 but for widgets using remote and remote filtered dataviews.

### CODE REVIEW

File: src/lib/viz/dataview/mode/DataViewRemote.ts [hunk 1]
```diff
@@ -169,18 +200,21 @@ export class DataViewRemote extends DataViewMode {
...
+    const filterOptions = options.filterId ? [options.filterId] : [];
+
+    const sql = this.sqlSource.getSQLWithFilters(filterOptions);
+    this.dataviewsApi.setSource(sql);
+  }
+}
 
+function getSource(dataSource: Layer | Source) {
   if (dataSource instanceof Source) {
     // TODO what about the other sources?
```
Comment (VictorVelarde) [inline src/lib/viz/dataview/mode/DataViewRemote.ts]: Is this comment still relevant?

**Review by VictorVelarde — APPROVED** [review]
LGTM 🚀 . Nice job!

**Comment by VictorVelarde** [comment]
Any blocker to merge this one? We should also add a small note to CHANGELOG

**Comment by jesusbotella** [comment]
I thought that we had already merged this PR the other day when releasing alpha version, but it seems like we didn't.

**Comment by VictorVelarde** [comment]
Duplicated

**Pull request closed without merging**